<launch>

    <node name="simpl_adaptive_super_twisting" pkg="sensors" type="simpl_adaptive_super_twisting" >
    <param name = "k_u" value = "0.1" />
	<param name = "k_psi" value = "0.1" />
	<param name = "kmin_u" value = "0.01" />
	<param name = "kmin_psi" value = "0.01" />
	<param name = "lambda_u" value = "0.001" />
	<param name = "lambda_psi" value = "1" />
    </node>

</launch>
